fe/fe-core/src/main/java/org/apache/doris/mtmv/MTMVPartitionUtil.java

```diff
@@ -426,6 +428,15 @@ private static boolean isSyncWithAllBaseTables(MTMVRefreshContext context, Strin
         return true;
     }
 
+    public static boolean isTableExcluded(Set<TableName> excludedTriggerTables, TableName tableNameToCheck) {
+        for (TableName tableName : excludedTriggerTables) {
+            if (tableName.like(tableNameToCheck)) {
```

Comment:
maybe like is not appropriate impl in TableName, it is better to impl in this Util class, because we could not use it in other places